src/components/Layout/Header/Header.tsx

```diff
@@ -186,39 +201,41 @@ export const Header = memo(() => {
         transitionProperty='all'
         transitionTimingFunction='cubic-bezier(0.4, 0, 0.2, 1)'
         top={0}
-        paddingTop={{ base: isDemoWallet ? 0 : 'env(safe-area-inset-top)', md: 0 }}
+        paddingTop={paddingTopProp2}
       >
         <HStack height='4.5rem' width='full' px={4}>
-          <HStack width='full' margin='0 auto' px={{ base: 0, xl: 4 }} spacing={0} columnGap={4}>
-            <Box flex={1} display={{ base: 'block', md: 'none' }}>
+          <HStack width='full' margin='0 auto' px={pxProp} spacing={0} columnGap={4}>
+            <Box flex={1} display={displayProp}>
               <IconButton aria-label='Open menu' onClick={onToggle} icon={<HamburgerIcon />} />
             </Box>
 
-            <Box display={{ base: 'block', md: 'none' }} mx='auto'>
+            <Box display={displayProp} mx='auto'>
               <AppLoadingIcon />
             </Box>
 
             <Flex
               justifyContent='flex-end'
               alignItems='center'
-              width={{ base: 'auto', md: 'full' }}
+              width={widthProp}
               flex={1}
               rowGap={4}
               columnGap={2}
             >
               <GlobalSeachButton />
-              {isDegradedState && <DegradedStateBanner />}
-              {isWalletConnectToDappsV2Enabled && (
-                <Box display={{ base: 'none', md: 'block' }}>
+              {isLargerThanMd && isDegradedState && <DegradedStateBanner />}
+              {isLargerThanMd && isWalletConnectToDappsV2Enabled && (
+                <Box display={displayProp}>
```

Comment:
We need a `displayProp2`, see:
```typescript
const displayProp = { base: 'block', md: 'none' }
```
but this should be
```typescript
const displayProp2 = { base: 'none', md: 'block' }
```
```suggestion
                <Box display={displayProp2}>
```

Reply:
47d972a9b7cca742ee17b0bdaec7c9df31095da5